bpm/bonita-util/bonita-common-util/src/test/java/org/bonitasoft/engine/bdm/model/BusinessObjectTest.java

```diff
@@ -191,4 +191,35 @@ public void getField_should_return_null_when_no_field_exists_for_the_given_name(
         assertThat(field).isNull();
     }
 
+    @Test
+    public void containsQuery_should_return_true_when_query_exists() throws Exception {
+        //given
+        final BusinessObject businessObject = aBO("aBo").withField(aBooleanField("field1"))
+                .withQuery(new Query("query1", null, null))
+                .withQuery(new Query("query2", null, null))
+                .build();
+        businessObject.setQualifiedName("com.company.model.Employee");
+
+        //when
+        boolean containsQuery = businessObject.containsQuery("query2");
+
+        //then
+        assertThat(containsQuery).isTrue();
+    }
+
+    @Test
+    public void containsQuery_should_return_false_when_query_does_exists() throws Exception {
```

Comment:
Method name should be **containsQuery_should_return_false_when_query_does_*not*_exists**